src/compute/compute_properties.cpp

```diff
@@ -11,9 +11,9 @@ void compute_number_of_elements_per_medium(
     const int nspec, const specfem::compute::mesh_to_compute_mapping &mapping,
     const specfem::mesh::tags<specfem::dimension::type::dim2> &tags,
     const specfem::kokkos::HostView1d<specfem::element::medium_tag>
-        &h_element_types,
+        &h_medium_tags,
```

Comment:
Would be nice to create a element_types constructor where these values are generated